app/controllers/users_controller.rb

```diff
@@ -104,14 +104,19 @@ def remove_association
   end
 
   def onboarding_update
-    current_user.assign_attributes(params[:user].permit(:summary, :location, :employment_title, :employer_name, :last_onboarding_page)) if params[:user]
+    if params[:user]
+      current_user.assign_attributes(params[:user].permit(:summary, :location, :employment_title, :employer_name, :last_onboarding_page))
+    end
     current_user.saw_onboarding = true
     authorize User
     render_update_response
   end
 
   def onboarding_checkbox_update
-    current_user.assign_attributes(params[:user].permit(:checked_code_of_conduct, :checked_terms_and_conditions, :email_membership_newsletter, :email_digest_periodic)) if params[:user]
+    if params[:user]
+      current_user.assign_attributes(params[:user].permit(:checked_code_of_conduct, :checked_terms_and_conditions, :email_membership_newsletter, :email_digest_periodic))
```

Comment:
this line is too long as well, what about something like?

```suggestion
      user_params = %i[
        checked_code_of_conduct checked_terms_and_conditions email_membership_newsletter email_digest_periodic
      ]
      current_user.assign_attributes(params[:user].permit(user_params))
```